python-backend/app/api/mines/mine/models/mine.py

```diff
@@ -183,12 +183,14 @@ def validate_mine_name(self, key, mine_name):
 
     @validates('mine_note')
     def validate_mine_note(self, key, mine_note):
+        mine_note = mine_note if mine_note else ''
```

Comment:
So the validators return the data back to database. I changed it to be empty string on None. Because . the database . has a NOT NULL constraint on mine_note 🤷‍♂️

Reply:
Sounds like the NOT NULL constraint should be removed, but this will work for now.